### PR TITLE
ts strict part 8: actions-tsconfig-constants

### PR DESCRIPTION
**Pull request by chrisnojima**
Please see https://github.com/keybase/client/pull/18383 for an overview

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/tsconfig.base.json [hunk 1]
```diff
@@ -9,9 +9,15 @@
     "jsx": "preserve",
     "module": "commonjs",
     "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```
Comment (buoyad) [inline shared/tsconfig.base.json]: Does this override `strict`?

Comment (chrisnojima) [reply shared/tsconfig.base.json]: strict is a bag of flags. we aren't anywhere close to supporting any
```Enable all strict type checking options. 
Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.```

Comment (chrisnojima) [reply shared/tsconfig.base.json]: the goal is to turn them all on eventually but we can't do them all at the same time and get it working across all files at the same time

**Pull request merged**